airbyte-integrations/connectors/destination-chroma/destination_chroma/destination.py

```diff
@@ -31,26 +24,18 @@
 
 BATCH_SIZE = 128
 
-embedder_map = {
-    "openai": OpenAIEmbedder,
-    "azure_openai": AzureOpenAIEmbedder,
-    "cohere": CohereEmbedder,
-    "fake": FakeEmbedder,
-    "from_field": FromFieldEmbedder,
-    "no_embedding": NoEmbedder,
-}
-
 
 class DestinationChroma(Destination):
 
     indexer: Indexer
     embedder: Embedder
 
     def _init_indexer(self, config: ConfigModel):
-        if config.embedding.mode == "azure_openai" or config.embedding.mode == "openai":
-            self.embedder = embedder_map[config.embedding.mode](config.embedding, config.processing.chunk_size)
-        else:
-            self.embedder = embedder_map[config.embedding.mode](config.embedding)
+        self.embedder = (
+            create_from_config(config.embedding, config.processing)
+            if config.embedding.mode != "no_embedding"
+            else NoEmbedder(config.embedding)
+        )
```

Comment:
Could this logic be added into the `create_from_config()` decision matrix? Not a blocker, just want to get your thoughts.